sys/dev/athn/if_athn_usb.c

```diff
@@ -24,6 +24,7 @@
 // #include "bpfilter.h"
 #define NBPFILTER 0 // TODO ?
 
+#include <dev/usb/usb.h>
```

Comment:
Why do we need this?

Reply:
For the definition of M_USBDEV. I checked that it's included by other USB-related drivers in FreeBSD.

Reply:
Ok so let's make a note that it also fixes M_USBDEV issue (in the commit message for example).